src/main/scala/Application.scala

```diff
@@ -1,18 +0,0 @@
-import scala.annotation.tailrec
-import scala.concurrent.duration._
-
-class Application extends App {
-
-  @tailrec
-  final def retry[A](block: () => A,
```

Comment:
the function should be inside object instead off class, there is no reason to keep it inside class